src/common.cc

```diff
@@ -120,29 +120,31 @@ WabtResult wabt_read_file(const char* filename,
                           size_t* out_size) {
   FILE* infile = fopen(filename, "rb");
   if (!infile) {
-    fprintf(stderr, "unable to read file: %s\n", filename);
+    char msg[128];
```

Comment:
Or (if you're going to change the line anyway) you could make it at least `PATH_MAX` + the size of the string below, so it's at least robust (against long paths) with the code as-is.